proposed/2023/vs-filters.md

```diff
@@ -0,0 +1,102 @@
+# Visual Studio Filters & Sorting Options
+
+- [Jon Douglas](https://github.com/JonDouglas)
+- 2023-06-23
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+This proposal represents a common ask for more filters and sorting options in the NuGet Package Manager experience in Visual Studio. 
+
+## Motivation 
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+As NuGet continues to expand as a package ecosystem, there has become a dire need for being able to filter and sort packages through one's daily management in their IDE of choice. While NuGet.org supports a plethora of filters and sorts on the search experience today, we'd like to bring these similar experiences to the various tabs(browse, installed, etc) in Visual Studio to help aid various jobs-to-be-done when it comes down to package management.
+
+A recent survey in May 2023 suggests that over 31% of people do not feel like there are adequate sorting and filtering options in Visual Studio when asked. Common free-response to this question relate around the following:
+
+- "There are no filtering options"
+- "More search options would make this better"
+- "Would like to find packages by tags, authors, licenses, and more"
+- "There are no way to sort packages outside of creation date"
+- "I'm not aware of any filters but prerelease on/off"
+- "Often I'm looking for a widely-used and respected package and I have to filter and sort through random niche-market packages"
+- "I should be able to filter based on an official package source and license of a package"
+- "I manually have to use google or the console because desired results are too hard to find without filters"
+- "Filters and sorts do not match the website"
+
+## Explanation
+
+### Functional explanation
+
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or pseudocode to show how this proposal would look. -->
+
+When a developer is using Visual Studio to search and manage NuGet packages, they will see a new filter and sort experience. 
+
+The browse filter experience will allow the developer to filter results based on common concepts such as "frameworks", "prerelease", "owners", "download count", "license", and "created on". 
```

Comment:
Do we know what will happen or what customers expect to happen for embedded licenses?

Reply:
license support will only be for properly expressed licenses. anything else would likely not work with the filter. data suggests this is a very rare case too.

Reply:
This is good rationale as to why: https://github.com/aaronpowell/dotnet-delice#undetermined-licenses and follows my thinking.